Tests/XcodeGenKitTests/ProjectSpecTests.swift

```diff
@@ -205,3 +216,9 @@ fileprivate func expectValidationError(_ project: Project, _ expectedError: Spec
     }
     throw failure("Supposed to fail with \"\(expectedError)\"", file: file, line: line)
 }
+
+fileprivate func expectMinimumXcodeGenVersionError(_ project: Project, minimumVersion: Version, xcodeGenVersion: Version, file: String = #file, line: Int = #line) throws {
```

Comment:
Can this just go inside the `it` closure, as it's only used there

Reply:
Sounds good.